apisix/core/config_local.lua

```diff
@@ -65,7 +65,10 @@ function _M.local_conf(force)
     end
 
     -- fill the default value by the schema
-    schema.validate(default_conf)
+    local ok, err = schema.validate(default_conf)
+    if not ok then
+        ngx.log(ngx.ERR, "======================== levy, err: ", err)
```

Comment:
levy?

Reply:
It will be changed to error(),  I just want to find out how many test cases will failed and fix them.

Reply:
I have changed this pr to draft.

Reply:
Done